tests/test_docs.py

```diff
@@ -0,0 +1,32 @@
+import doctest
+import importlib
+import os
+import unittest
+
+import skmatter
+
+
+print(doctest.NORMALIZE_WHITESPACE)
+
+
+class TestDoctests(unittest.TestCase):
```

Comment:
this was a bit flaky on equsitore ...

It might be better to use pytest just as a test runner for doctests, even if it is not used anywhere else

Reply:
Hmm if I remember correctly I tried pytest but it did not detect the tests because they are hidden in files starting `_`...

But, yes I can try it again.

Reply:
Seems to work. I don't know what I did wrong before.